### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2023-11-02

### DISCUSSION THREAD

**Discussion by ivanperez-keera** [space-ros/space-ros#96] [General]
Previous meeting: #95

# Agenda

- [x] Introduce any new attendees.
- [x] Discuss published release humble-2023.10.0, lessons learned (lead @Bckempa ).
- [x] Open next meeting agenda discussion

**Comment by ivanperez-keera** [comment]
@EzraBrooks I've gone ahead and set up the agenda for today just so that we have something. I hope that's ok. Feel free to remove/edit as needed if I've done something not the way you wanted it.

**Reply by EzraBrooks** [reply]
Thanks - I keep meaning to set a reminder for myself and create a template for these, but I keep getting immediately distracted by project work after the meeting ends.

**Reply by Bckempa** [reply]
I added an item to the agenda to open the next meeting issue at the end

**Comment by Bckempa** [comment]
If you have time and want to contribute, take a look at the issues [tagged "release followup"](https://github.com/space-ros/space-ros/labels/release%20followup) and provide any feedback, triaging, or replication reports you are able to. Even just thoughts and comments are appreciated.

**Reply by Bckempa** [reply]
Also, to nominate issues for the next release add the `humble-2024.02.0` milestone to the issue (note milestones are listed below labels on the right side of the issue page)

**Comment by EzraBrooks** [comment]
Next meeting: #113
